message/validation/partial_validation.go

```diff
@@ -104,6 +104,12 @@ func (mv *messageValidator) validatePartialSignatureMessageSemantics(
 	}
 
 	for _, message := range partialSignatureMessages.Messages {
+		// MSGVALIDATIONREVIEW: add signature size check as below:
```

Comment:
Being fixed here: https://github.com/ssvlabs/ssv/pull/1488

Reply:
It has `ssz-size:"96"` assertion, so `ErrWrongBLSSignatureSize` will never be returned. However, I don't mind adding a comment

Reply:
Oh true. Thanks! I'll drop the check and leave a comment